var/spack/repos/builtin/packages/flux-core/package.py

```diff
@@ -39,7 +41,7 @@ class FluxCore(AutotoolsPackage):
     depends_on("czmq")
     depends_on("czmq@2.2:3.99", when="@0.1:0.6")
     depends_on("czmq@3.0.1:", when="@0.7:")
-    depends_on("hwloc@1.11.1:1.99")
+    depends_on("hwloc@1.11.1:")
```

Comment:
Was the constraint wrong before? Or do we need to maintain it for older releases and relax it only for newer ones?

Reply:
I should probably add a when since I only verified with a fairly recent flux-core version.

I'll poke the actual flux devs to see how safe that is

Reply:
Verified on mattermost that ```flux-core@0.17``` is when ```hwloc@2:``` became supported